src/share/classes/com/sun/javatest/regtest/exec/RegressionScript.java

```diff
@@ -135,7 +135,7 @@ public Status run(String[] argv, TestDescription td, TestEnvironment env) {
             hostname = "127.0.0.1";
         }
         testResult.putProperty("hostname", hostname);
-        String[] props = { "user.name" };
+        String[] props = { "user.name", "os.name", "os.arch", "os.version" };
         for (String p: props) {
             testResult.putProperty(p, System.getProperty(p));
         }
```

Comment:
Nice and simple ... but probably too simple.

This will report the properties for the JVM running `jtreg`, which is OK for `user.name`, but for the `os.*` properties we probably want the values for the test JVM.  You can get the properties from `params.getTestJDK().getProperties()` ... or better still, use/report the values cached in `params.getTestOS()`.
 And, if we're going as far as using `params.getTestOS()` it might be worth printing out the full `OS.toString()`.
 
 So, suggest either
 
 *  `testResult.putProperty(` _name_ `, os.` _field_ `)` for the three values, or else just
 * `testProperty.putProperty("testJDK_OS", os.toString())`
 
 (or both?)

Reply:
Understood.

Does make sense to also keep the additional information report for the JVM running `jtreg`? In summary, this would introduce 8 new lines, like:

```
(JVM running jtreg) JDK_OS = ...
(JVM running jtreg) os.arch = ...
(JVM running jtreg) os.name = ...
(JVM running jtreg) os.version = ...
(Test JVM) JDK_OS = ...
(Test JVM) os.arch = ...
(Test JVM) os.name = ...
(Test JVM) os.version = ...
```

Reply:
It's less important/interesting to give details about the VM running jtreg itself.

The primary interest is the VM running the test, because these `os.*` properties can be used to filter test execution.